tests/ha/check_hawk.pm

```diff
@@ -51,6 +51,11 @@ sub run {
 
     barrier_wait("HAWK_CHECKED_$cluster_name");
 
+    # This module is the last one scheduled in cluster verification migration tests. Since node one
+    # handles the barriers when not using support server, we need to give it more time for the other
+    # nodes to finish
+    sleep bmwqemu::scale_timeout(10) if (is_node(1) and get_var('TEST') =~ /verify/ and !get_var('USE_SUPPORT_SERVER'));
```

Comment:
It's not better to add this `sleep` at the end of the file? Hawk GUI is not tested with migration but may be possible later, no?

Other than this, LGTM!

Reply:
Yes, I think it'll look better. I'll change it.

It's the same though, as we're not testing hawk in upgrade scenarios.

Reply:
> It's the same though, as we're not testing hawk in upgrade scenarios.

Here yes, it's just for the future ;-)